scholly/service/core/data.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   REACT_APP_TEAM_INFO: 'REACT_APP_TEAM_INFO',
   REACT_APP_TEAM_MEMBERS: 'REACT_APP_TEAM_MEMBERS',
   REACT_APP_TEAM_HOMEPAGE: 'REACT_APP_TEAM_HOMEPAGE',
-  // Array containing names of pages that the client wants deployed
-  REACT_APP_WEB_PAGES: 'REACT_APP_WEB_PAGES',
+  // Contains array of pages to deploy, and client layout selection for the entire website and for publications list
+  REACT_APP_WEB_METADATA: 'REACT_APP_WEB_METADATA',
```

Comment:
Prefer prefixing with TEAM_* to 1: maintain consistent with the other Globals and to make it more meaningful. Just web metadata doesn’t make a lot of sense imo

Reply:
What about “TEAM_SITE_METADATA”?

Reply:
sounds good